explorer/src/migration/1595458073813-RecreateJobRunSearchableAddressesIndex.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class RecreateJobRunSearchableAddressesIndex1595458073813
```

Comment:
Could we import the old class in here and reexport it under a different name? Could DRY it up that way.